src/incubator/f5-bigip-ingress/README.md

```diff
@@ -1,12 +1,25 @@
-# alpha status
+# Helm Chart for using the F5 BIG-IP as an ingress
 
-This chart is in an alpha status. It can be used to deploy an ingress with on simple service. 
+This chart simplifies repeatable, versioned use of the F5 BIG-IP as an [ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) within Kubernetes or OpenShift by using the [k8s-bigip-ctlr](http://clouddocs.f5.com/containers/latest/kubernetes/kctlr-ingress.html) as an ingress controller.
 
-Note that your [F5 k8s-bigip-ctlr](https://github.com/F5Networks/k8s-bigip-ctlr) should already be running in your cluster.
+## Requirements
 
-# Suggested Use
+The chart presumes:
+- You have [Helm with Tiller](https://docs.helm.sh/using_helm/#installing-helm) installed on your cluster with appropriate permissions.
+- You have the k8s-bigip-ctlr already running in your cluster. You can use a separate chart - the [f5-bigip-ctlr chart](https://github.com/F5Networks/charts/tree/master/src/stable/f5-bigip-ctlr) - to deploy the controller, or you can deploy it [manually](http://clouddocs.f5.com/containers/latest/kubernetes/kctlr-app-install.html). 
+- The services accepting traffic from the ingress are configured separately.
 
-Copy the `values.yaml` file and use it as the basis for your own ingress specification and then pass your custom values file to:
+> Note - This chart and the [f5-bigip-controller](https://github.com/recursivelycurious/charts/tree/wip/src/incubator/f5-bigip-ctlr) chart can be used independently or together. If you or your organization author your own charts either or both may be used as a subchart.
+>
+> Similarly, this ingress chart can be combined -- either as a parent chart or a subchart -- with charts that define the services accepting traffic.
+
+## Chart Details
+
+The chart creates an Ingress resource for use with the [k8s-bigip-ctlr](http://clouddocs.f5.com/containers/latest/kubernetes/index.html).
+
+## Installing the Chart
+
+Copy the `values.yaml` file and use it as the basis for your own ingress specification and then pass your custom values file when invoking `helm install`:
```

Comment:
1. Copy the values file. Use this file as the basis for your own Ingress specification.
2. Pass your custom values file when running helm install, as shown in the example below.

Reply:
Done.